bin/container_daemon

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# this file requires sh over bash:
+set -e
+user=textile
+repo="$THREADS_REPO"
+
+if [ $(id -u) -eq 0 ]; then
+	echo "Changing user to $user"
+	# ensure folder is writable
+	su-exec "$user" test -w "$repo" || chown -R -- "$user" "$repo"
+	# restart script with new privileges
+	exec su-exec "$user" "$0" "$@"
+fi
+
+# if the first argument is daemon
+if [ "$1" = "daemon" ]; then
+	shift
+fi
+
+exec threads "$@"
```

Comment:
I don't see how this would work since there's no executable named `threads`, right?

Reply:
it moved, https://github.com/textileio/go-textile-threads/pull/169/files#diff-3254677a7917c6c01f55212f86c57fbfR44

Reply:
ah ok 👍